doc/man3/OSSL_PROVIDER.pod

```diff
@@ -66,6 +69,10 @@ runs its teardown function.
 OSSL_PROVIDER_available() checks if a named provider is available
 for use.
 
+OSSL_PROVIDER_do_all() iterates over all loaded providers, calling
```

Comment:
How about OSSL_PROVIDER_iterate_for_all or something?

the name ..._do_all() reminds me of the magic function
```C
 void do_what_i_mean(void);
```

Reply:
but if you like _do_all I am okay with it as well :smile:

Reply:
`_do_all` is an already existing pattern